python/pfs/drp/stella/createDetGeom.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import argparse
+
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        super(CreateDetGeomTask, self).__init__(*args, **kwargs)
+
+    def run(self):
+        gain = (1.24, 1.24, 1.27, 1.18, 1.26, 1.20, 1.24, 1.26)
+        rdnoise = (3.61, 3.78, 3.18, 2.95, 3.19, 3.80, 4.51, 3.18)
+        
+        outDir = os.getenv('OBS_PFS_DIR', '~/stella_git/obs_pfs')
+        print 'outDir = <',outDir,'>'
+        outDir = outDir+'/pfs/camera'
```

Comment:
The output directory will not change so I removed it from the parameter list.
You are right, this should be an obs_pfs task. Originally it was an ipython notebook in drp_stella, before obs_pfs was created. Now it should really go into obs_pfs, so I moved both createDetGeom.py (bin.src and python/pfs/drp/stella) to obs_pfs (bin.src and python/lsst/obs/pfs)